Python/Fibbonacci.py

```
@@ -0,0 +1,9 @@
+def fibonacci(n):
+    seq = [0, 1]
+    for i in range(2, n):
```

Ruff (0.14.1)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)